packages/slice-machine/src/features/environments/useActiveEnvironment.ts

```diff
@@ -1,7 +1,15 @@
-import { useRequest } from "@prismicio/editor-support/Suspense";
+import {
+  Fetcher,
+  invalidateFetcherData,
+  useRequest,
+} from "@prismicio/editor-support/Suspense";
 
 import { getActiveEnvironment } from "./actions/getActiveEnvironment";
 
+export function invalidateActiveEnvironmentData() {
+  invalidateFetcherData(getActiveEnvironment as Fetcher);
```

Comment:
This cast should not be needed, but `invalidateFetcherData` types are causing this error:

<img width="969" height="281" alt="Image" src="https://github.com/user-attachments/assets/139fde97-2dff-46e3-ab27-b9b8655d591c" />


`invalidateFetcherData` should not really care about `args`, so the types might need to be adjusted.

Reply:
Hum here I think it could be a problem. `invalidateFetcherData` actually can care about `args`.
We need to dig that up a bit as I remember our custom useRequest was not supporting all use case.

If you check:

```
ARGS extends readonly SerializableArg[]
``

So here we pass an object and I think useRequest don't support this with an object being passed.